actions/admin/contact_old.go

```diff
@@ -0,0 +1,216 @@
+package admin
+
+import (
+	"net/http"
+
+	"github.com/bitcoin-sv/spv-wallet/actions/common"
+	"github.com/bitcoin-sv/spv-wallet/engine"
+	"github.com/bitcoin-sv/spv-wallet/engine/spverrors"
+	"github.com/bitcoin-sv/spv-wallet/mappings"
+	"github.com/bitcoin-sv/spv-wallet/models"
+	"github.com/bitcoin-sv/spv-wallet/models/filter"
+	"github.com/bitcoin-sv/spv-wallet/server/reqctx"
+	"github.com/gin-gonic/gin"
+)
+
+// contactsSearchOld will fetch a list of contacts filtered by Metadata and ContactFilters
+// Search for contacts filtering by metadata and ContactFilters godoc
+// @DeprecatedRouter /v1/admin/contact/search [post]
+// @Summary		Search for contacts
+// @Description	Search for contacts
+// @Tags		Admin
+// @Produce		json
+// @Param		SearchContacts body filter.SearchContacts false "Supports targeted resource searches with filters and metadata, plus options for pagination and sorting to streamline data exploration and analysis"
+// @Success		200 {object} models.SearchContactsResponse "List of contacts"
+// @Failure		400	"Bad request - Error while parsing SearchContacts from request body"
+// @Failure 	500	"Internal server error - Error while searching for contacts"
+// @Router		/v1/admin/contact/search [post]
+// @Security	x-auth-xpub
+func contactsSearchOld(c *gin.Context, _ *reqctx.AdminContext) {
+	logger := reqctx.Logger(c)
+	engine := reqctx.Engine(c)
+	var reqParams filter.SearchContacts
+	if err := c.Bind(&reqParams); err != nil {
+		spverrors.ErrorResponse(c, spverrors.ErrCannotBindRequest, logger)
```

Comment:
☝️

Reply:
same old code, so been moved to _old.go and i left untouched.